### PR TITLE
Set MAX_CONTENT_LENGTH from environment

### PR DESCRIPTION
**Pull request by pontus**
Tell flask to allow larger bodies by setting`MAX_CONTENT_LENGTH` from the environment.

### CODE REVIEW

**Comment by pragermh** [comment]
Thanks, but when I try to upload files larger than MAX_CONTENT_LENGTH, the app shows a ERR_CONNECTION_RESET page rather than the general error page.

**Comment by pontus** [comment]
I didn't verify that behaviour, but believe this *should be* https://stackoverflow.com/questions/19911106/flask-file-upload-limit/66243421#66243421, although it doesn't seem to behave as expected when running in development.

I'm not sure if this is a problem though since this will reasonably be in sync with the nginx configuration (which I believe should generate it's own error code), but I'm of course willing to research it if you think it's needed.

**Comment by pragermh** [comment]
Yes, you're right, run together with nginx it shows the expected error page.

File: .env [hunk 1]
```diff
@@ -27,7 +27,9 @@ CAS_SERVER=https://auth.biodiversitydata.se
 CAS_AFTER_LOGIN=main_bp.index
 
 # File upload
-MAX_CONTENT_LENGTH=838860800
+# Size limit is set in nginx (molecular.conf: client_max_body_size 800m;),
+# but also here, to show flask (rather than nginx) error page
```
Comment (pontus) [inline .env:31]: Technically, we need to tell nginx to allow larger uploads so it passes them through.

**Pull request merged**